src/turndown.js

```diff
@@ -421,6 +421,174 @@ turndownService.addRule("div_with_class", {
   }
 })
 
+turndownService.addRule("multiple_choice_question", {
+  filter: (node, options) => {
+    if (node.nodeName === "DIV" && node.getAttribute("class")) {
+      if (
+        node.getAttribute("class") === "problem_question" &&
+        node.getElementsByClassName("problem_radio_input")
+      ) {
+        return true
+      }
+    }
+    return false
+  },
+  replacement: (content, node, options) => {
+    const questionId = node.getAttribute("id")
+    let questionMarkdown = Array.from(
+      node.getElementsByTagName("fieldset")[0].childNodes
+    )
+      .filter(child => {
+        if (
+          child.nodeName === "DIV" &&
+          child.getAttribute("class").includes("choice") &&
+          child.getElementsByClassName("problem_radio_input")
+        ) {
+          return true
+        }
+      })
+      .map(choice => {
+        const isCorrect =
+          choice
+            .getElementsByClassName("problem_radio_input")[0]
+            .getAttribute("correct") === "true"
+        return `{{<choice isCorrect="${isCorrect}">}}${turndownService.turndown(
```

Comment:
Minor formatting nitpick:

Here and other places where you are calling shortcodes, can we make sure there are spaces on either side of the decorators every time?  I'm not sure how much this matters, but every place I look at in the Hugo documentation that shows shortcodes has a space after the opening decorator and before the closing.  So this one for example would be :

`{{< choice isCorrect="${isCorrect}" >}}`... etc.

Our code formatter isn't really capable of handling situations like this where we are defining code in strings like this.  It may be unnecessary, but this is the way I've always done it because it's the way I see shortcodes called in all of Hugo's documentation.  With closing tags it would be:

`{{< /choice >}}`

Examples of what I'm talking about can be found here: https://gohugo.io/templates/shortcode-templates/